docs/UserGuide.md

```diff
@@ -8,54 +8,126 @@ pageNav: 3
 
 EduTrack is a **desktop app for managing classes designed for tutors**, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). Whether you are managing multiple students or just need quick access to individual student data, EduTrack empowers you with effortless control and organization.
 
+If you are new to this guide, click [here](#about) to find out more about this user guide.
+
+If you are keen to get started on using EduTrack, click [here](#quick-start) for our quick start guide.
+
 --------------------------------------------------------------------------------------------------------------------
 
 ## Table of Contents
 
 1. [Quick start](#quick-start)
+2. [About](#about)
+   - [Structure of this user guide](#structure-of-this-user-guide)
+   - [Reading this user guide](#reading-this-user-guide)
+     - [Common icons](#common-icons)
+     - [Common terms](#common-terms)
+     - [Command Format](#command-format)
+     - [Command Parameters](#command-parameters)
 2. [Features](#features)
-    - [Viewing help : `help`](#viewing-help--help)
-    - [Adding a class : `add`](#adding-a-class--add)
-    - [Removing a class : `remove`](#removing-a-class--remove)
-    - [Viewing a class : `view`](#viewing-a-class--view)
-    - [Editing a class: `view`](#editing-a-class--edit)
-    - [Adding a student : `add`](#adding-a-student--add)
-    - [Removing a student : `remove`](#removing-a-student--remove)
-    - [Starting a lesson : `startlesson`](#starting-a-lesson--startlesson)
-    - [Marking a student present : `mark`](#marking-your-students-attendance-as-present--mark)
-    - [Marking a student absent : `unmark`](#marking-all-students-attendance-in-your-class-as-present--markall)
-    - [Marking all students present : `markall`](#marking-your-students-attendance-as-absent--unmark)
-    - [Exiting the program : `exit`](#exiting-the-program--exit)
-    - [Saving the data](#saving-the-data)
-    - [Adding a list of students : `coming in v1.3`](#adding-a-list-of-students--coming-in-v13)
-    - [Updating a student : `coming in v1.3`](#updating-a-student--coming-in-v13)
-    - [Adding a lesson : `coming in v1.3`](#adding-a-lesson--coming-in-v13)
-    - [Removing a lesson : `coming in v1.3`](#removing-a-lesson--coming-in-v13)
-    - [Finding a student : `coming in v1.3`](#finding-a-student--coming-in-v13)
+    - [Class Commands](#class-commands)
+      - [Adding a class : `add`](#adding-a-class-add)
+      - [Removing a class : `remove`](#removing-a-class-remove)
+      - [Viewing a class : `view`](#viewing-a-class-view)
+      - [Editing a class: `view`](#editing-a-class-edit)
```

Comment:
I noticed a small typo here, it should be `edit` instead, can you help fix it too thanks!